app/assets/stylesheets/modules/markers.css.scss

```diff
@@ -145,7 +145,7 @@
   width: 100px;
 }
 
-#highlightMarkerLayer img {
+img[src*="pulsing_location_marker"] {
```

Comment:
Looks for substring. For example it matches 'sssssssspulsing_location_markersssssss'

Reply:
I think "pulsating" is the right word

Reply:
I'm using the word that Michael used in the ticket. But I'm fine changing it if you prefer